auth/tenant_mgt.go

```diff
@@ -275,6 +282,11 @@ func (t *TenantToUpdate) EnableEmailLinkSignIn(enable bool) *TenantToUpdate {
 	return t.set(enableEmailLinkSignInKey, enable)
 }
 
+// EnableAnonymousUsers enables or disables anonymous user.
```

Comment:
Looks like `EnableAnonymousUsers` is a literal that should be backticked for code font.